components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/utils/LocalEntryClient.java

```diff
@@ -32,8 +33,10 @@
     private LocalEntryAdminServiceStub localEntryAdminServiceStub;
 
     static final String backendURLl = "local:///services/";
+    private String tenantDomain;
 
-    public LocalEntryClient() throws AxisFault {
+    public LocalEntryClient(String tenantDomain) throws AxisFault {
+        this.tenantDomain = tenantDomain;
         localEntryAdminServiceStub = new LocalEntryAdminServiceStub(null,
```

Comment:
When initialising the stub, better to pass the axis2 configuration context without passing null.

check https://github.com/wso2/carbon-apimgt/blob/6.x/components/apimgt/org.wso2.carbon.apimgt.keymgt.client/src/main/java/org/wso2/carbon/apimgt/keymgt/client/SubscriberKeyMgtClient.java#L48

Reply:
Fixed